package-lock.json

```diff
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "thecyberhub.org",
-      "version": "0.53.12",
+      "version": "0.53.14",
```

Comment:
```suggestion
      "version": "0.53.12",
```

we actually never push other files other than we have installed anything new in the packages 
can u fix this is package-lock.json

Reply:
Yes for sure

Reply:
> Yes for sure

if u need any help or having any doubts, let us know